### PR TITLE
Use distutils find_executable instead of which

### PR DESCRIPTION
**Pull request by puiterwijk**
Fixes: #44

Signed-off-by: Patrick Uiterwijk <patrick@puiterwijk.org>

### CODE REVIEW

**Review by AlanCoding — APPROVED** [review]

File: awx/plugins/ansible_inventory/backport.py [hunk 1]
```diff
@@ -314,12 +315,17 @@ def format_group(group):
         return format_group(top)
 
 
+def which(binary):
```
Comment (ryanpetrello) [inline awx/plugins/ansible_inventory/backport.py]: distutils has a utility that does basically this (which we may prefer to use):

from distutils.spawn import find_executable

**Review by ryanpetrello — CHANGES_REQUESTED** [review]

File: awx/plugins/ansible_inventory/backport.py (hunk 1 above)
Comment (puiterwijk) [reply awx/plugins/ansible_inventory/backport.py]: So that means you want to always keep the dep on distutils? Okay.

Comment (wwitzel3) [reply awx/plugins/ansible_inventory/backport.py]: I was going to suggest distutils or whole out copying the python3 shutil implementation (https://github.com/python/cpython/blob/master/Lib/shutil.py#L1096).

**Review by wwitzel3 — APPROVED** [review]

**Review by ryanpetrello — APPROVED** [review]

**Comment by cchurch** [comment]
Should we also update https://github.com/ansible/ansible-inventory-backport to use this approach instead of https://github.com/ansible/ansible-inventory-backport/commit/8f9ae6a30ef65681e2aef50d345017542ee91bff?

**Comment by AlanCoding** [comment]
yes, let's do that

**Pull request merged**